clusterloader2/pkg/util/cluster.go

```diff
@@ -19,11 +19,12 @@ package util
 import (
 	"fmt"
 
+	"k8s.io/perf-tests/clusterloader2/pkg/framework/client"
```

Comment:
last nit - please revert this line - we try to keep imports in alphabetical order, so the previous one was good

Reply:
not the `go package->this project package->vendor package`?